src/net/sourceforge/kolmafia/textui/Parser.java

```diff
@@ -1,5 +1,7 @@
 package net.sourceforge.kolmafia.textui;
 
+import static org.eclipse.lsp4j.DiagnosticSeverity.*;
```

Comment:
Can we just write out DiagnosticSeverity.Error and DiagnosticSeverity.Warning in the two places that you're using this static import?

Reply:
...sure (._. )